_posts/2016-02-10-usando-PHP-7.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "Usando PHP 7"
+date:   2016-02-10 22:05:00
+author: 
+    name: "Marcio Albuquerque"
+    mail: "marcio.lima.albuquerque@gmail.com"
+    github: "mlalbuquerque"
+    blog: "http://culturabeta.com.br"
+    twitter: "@mlalbuquerque"
+    facebook: "mlalbuquerque"
+categories: experiência php7
+cover:  "/assets/posts/2016/02/PHP-7.png"
+---
+
+Estou usando PHP 7 tem um tempo já e só tenho uma coisa a falar...
+
+*__CURTINDO DEMAIS!!!__*
+
+Não vou escrever sobre as mudanças, até porque já existem vários posts sobre as novas funcionalidades.
+Só para citar alguns:
+
+* [New features](http://php.net/manual/pt_BR/migration70.new-features.php) - Site oficial do PHP com as novas funcionalidades
+* [What to Expect When You're Expecting: PHP 7, Part 1](https://blog.engineyard.com/2015/what-to-expect-php-7) - 
+A Engine Yard dando seus pitacos
+* [Getting Ready for PHP 7](https://www.digitalocean.com/company/blog/getting-ready-for-php-7/) - A Digital Ocean também comentando
+um sobre algumas novidades
+
+O que vou escrever é sobre como instalei e algumas das novidades que já estou usando em projetos pessoais. Mas antes, um adendo:
+a velocidade do PHP 7 comparado com o PHP 5.6 é notável. Rodei algumas aplicações pesadas e dá para notar a diferença na resposta.
+E isso sem ligar o cache (nas duas versões).
+
+Ok, como instalei o PHP 7 no Ubuntu 15.10? Veja abaixo:
+
+{% highlight shell %}
+sudo add-apt-repository ppa:ondrej/php
+sudo apt-get update
+sudo apt-get install php7.0
+{% endhighlight %}
+
+Se quiser saber que módulos podem ser instalados:
+
+{% highlight shell %}
+sudo apt-cache search php7.0
+{% endhighlight %}
+
+Para saber maiores detalhes, tirei as informações da [Digital Ocean](https://www.digitalocean.com/community/tutorials/how-to-upgrade-to-php-7-on-ubuntu-14-04).
```

Comment:
Entendo que não a intenção ser um post altamente complexo/to mas sim falando de suas experiências, mas acredito que nessa parte seria bom ao menos linkar um ou outro post sobre instalar em outras distribuições. Pode ser desnecessário no sentido de que se o cara tem interesse, ele vai procurar. Mas sendo o primeiro post do blog, tenho em mente que algo que aconchegue mais os leitores tenha um melhor retorno.